platform/linux-generic/odp_packet_io.c

```diff
@@ -581,10 +609,19 @@ int odp_pktio_config(odp_pktio_t hdl, const odp_pktio_config_t *config)
 	entry->s.in_chksums.chksum.sctp = config->pktin.bit.sctp_chksum;
 
 	entry->s.enabled.tx_ts = config->pktout.bit.ts_ena;
+	entry->s.enabled.tx_compl = config->pktout.bit.tx_compl_ena;
+
+	if (entry->s.enabled.tx_compl)
+		if (configure_tx_event_compl(entry)) {
+			ODP_ERR("Unable to configure Tx event completion\n");
+			res = -1;
+			goto out_unlock;
```

Comment:
Since code with goto does not save lines and it does print while holding the lock. It would be better to do the same sequence as if-statement above does:

unlock_entry(entry);
ODP_ERR();
return -1;

Reply:
Done in v2.